### ISSUE TITLE
potential bug

### ISSUE THREAD

**Issue by dotchen** [midas-research/man-sf-emnlp#2]
Here https://github.com/midas-research/man-sf-emnlp/blob/main/train.py#L107-L109,

Why is this `F.softmax`? should the right thing be `F.log_softmax`?

reference: https://pytorch.org/docs/1.9.0/generated/torch.nn.functional.nll_loss.html
